lib/clone.js

```diff
@@ -86,6 +87,16 @@ module.exports = internals.clone = function (obj, options = {}, _seen = null) {
             continue;
         }
 
+        // Can only be covered in node 21+
+        /* $lab:coverage:off$ */
+        if (internals.structuredCloneExists &&
+            baseProto === Types.error &&
+            key === 'stack') {
+
+            continue;       // Already a part of the base object
+        }
+        /* $lab:coverage:on$ */
```

Comment:
This could use the coverage-predicates feature of lab to allow it to be conditionally included in coverage checks. But the `if (internals.structuredCloneExists)` itself would still need to be disabled for coverage checks, since it will always either be `true` or `false`. It's probably overkill for a backport, though.

Reply:
Sure, I went for the shorter path since I consider the current main branch to be short lived. Hopefully we'll have the next major out soon and leave this one behind.